web/static/js/Plugin.jsx

```diff
@@ -23,22 +22,22 @@ var Plugin = React.createClass({
   },
 
   componentWillUnmount: function() {
-    $(this.getDOMNode()).find('[title]').tooltip('destroy');
+    $(findDOMNode(this)).find('[title]').tooltip('destroy');
   },
 
   componentDidUpdate: function() {
     this.addBootstrapTooltips();
   },
 
   addBootstrapTooltips: function() {
-    $(this.getDOMNode()).find('[title]').tooltip({
+    $(findDOMNode(this)).find('[title]').tooltip({
       animation: false,
       container: 'body'
     });
   },
 
   goToDetailsPage: function() {
-    transitionTo("plugin", {slug: this.props.plugin.slug});
+    browserHistory.push("plugin/" + this.props.plugin.slug);
```

Comment:
It is entirely possible that I'm missing something. From what I could find, this is expected to either be a string (path), or a [`LocationDescription`](https://github.com/mjackson/history/blob/master/docs/Glossary.md#locationdescriptor), and I didn't see any way to declare route wildcards in a LocationDescription.